src/Claimer.sol

```diff
@@ -115,12 +117,19 @@ contract Claimer is Multicall {
       revert ClaimArraySizeMismatch(_winners.length, _prizeIndices.length);
     }
 
-    uint96 feePerClaim = SafeCast.toUint96(
-      _computeFeePerClaimForBatch(_tier, _winners, _prizeIndices)
-    );
+    uint96 feePerClaim;
 
-    if (feePerClaim < _minVrgdaFeePerClaim) {
-      revert VrgdaClaimFeeBelowMin(_minVrgdaFeePerClaim, feePerClaim);
+    /**
+     * If the claimer hasn't specified both a min fee and a fee recipient, we assume that they don't
+     * expect a fee and save them some gas on the calculation.
+     */
+    if (_feeRecipient != address(0) || _minVrgdaFeePerClaim != 0) {
```

Comment:
I don't think you need the ``|| _minVrgdaFeePerClaim``

if address is zero and the min vrgda fee is non-zero this code block shouldn't run.

Reply:
Yes because of this condition in the PrizePool: https://github.com/GenerationSoftware/pt-v5-prize-pool/blob/9c950bcfc58ff9d28c882ba985b8c276805e25e2/src/PrizePool.sol#L428

But if we allowed the fee to be burnt, we could still calculate it here.
Maybe the condition in the PrizePool should be moved here.

Reply:
The logic behind my choice here was that if a caller sends a min fee, but sets the fee recipient address to the zero address, the call should revert. Since it already reverts in the Prize Pool with these conditions, it was important that a claim fee be set if the min fee was non-zero.

If we instead check for this reversion in the claimer like @PierrickGT is suggesting, it would save some gas in this case and we could clean up the if-statement logic as well.